backends/graphs/graphs.cpp

```diff
@@ -55,6 +55,116 @@ MidEnd::MidEnd(CompilerOptions& options) {
     });
 }
 
+class Graphs::GraphAttributeSetter {
```

Comment:
I personally think that the `main` function for the backend and the `Graph` methods implementation belong in 2 different files; apart from that it looks ok.

Reply:
created a new file "p4c-graphs.cpp" to accommodate `main` funcion.